src/webapis/WebAPIs.ts

```diff
@@ -163,4 +180,15 @@ export class WebAPIs {
             response.status(503);
         }
     }
+
+    async handleRuleserverUpdate(ruleserver: Ruleserver, { since, request, response }: { since: string, request: express.Request, response: express.Response }) {
+        // TODO Have to do this because express sends keep alive by default and during tests
```

Comment:
I don't think it is a `TODO`, right? It's a workaround for an oddity in the library we're using.

Reply:
it's a todo because I don't know if this is going to be a problem when someone is polling every second for rules in production

Reply:
I don't get it. Usually, `TODO` means that you haven't done it yet, just like `FIXME`. Is this what you have in mind?